tests/e2e/machine_pool_test.go

```diff
@@ -71,5 +69,50 @@ var _ = Describe("TF Test", func() {
 				Expect(respBody.ID()).To(Equal(mp_out.Name))
 			})
 		})
+		Context("Author:amalykhi-High-OCP-64905 @OCP-64905 @amalykhi", func() {
+			It("Edit/delete second machinepool labels", func() {
```

Comment:
Let's make It spec to be case unit and it contains the title.
Format like 
```
It("Author:amalykhi-High-OCP-64905 Edit/delete second machinepool labels", func(){
<steps>
})
```
In future we can integrate the result to report portal and at the same time we can parse the result to send it to slack